controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -187,6 +187,18 @@ spec:
       for: 1m
       labels:
         severity: critical
+    - alert: CephOSDDown
+      annotations:
+        description: Ceph OSD(s) is/are down, please check the ceph-osd daemons and take corrective measures. Please see runbook URL for more details
+        message: An OSD has been marked down
+        severity_level: warning
+        storage_type: ceph
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephOSDDown.md
+      expr: |
+        ceph_health_detail{name="OSD_DOWN"} == 1
```

Comment:
If there is more than one OSD down, this alert will still fire, right? Checking for `1` is just that there is one health warning regardless of the number of OSDs down, correct?

Reply:
yes true.